collector/mssql.go

```diff
@@ -246,1021 +244,1026 @@ type MSSQLCollector struct {
-			prometheus.BuildFQName(Namespace, subsystem, "log_pool_requests_old_vlf_persec"),
-			"(LogPoolRequestsOldVLFPersec)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_pool_requests_old_vlf"),
+			"(Databases.LogPoolRequestsOldVLFPersec)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		LogPoolRequestsPersec: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "log_pool_requests_persec"),
-			"(LogPoolRequestsPersec)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_pool_requests"),
+			"(Databases.LogPoolRequestsPersec)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		LogPoolTotalActiveLogSize: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "log_pool_total_active_log_size"),
-			"(LogPoolTotalActiveLogSize)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_pool_total_active_log_size"),
+			"(Databases.LogPoolTotalActiveLogSize)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		LogPoolTotalSharedPoolSize: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "log_pool_total_shared_pool_size"),
-			"(LogPoolTotalSharedPoolSize)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_pool_total_shared_pool_size"),
+			"(Databases.LogPoolTotalSharedPoolSize)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		LogShrinks: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "log_shrinks"),
-			"(LogShrinks)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_shrinks"),
+			"(Databases.LogShrinks)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		LogTruncations: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "log_truncations"),
-			"(LogTruncations)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_log_truncations"),
+			"(Databases.LogTruncations)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		PercentLogUsed: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "percent_log_used"),
-			"(PercentLogUsed)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_percent_log_used"),
+			"(Databases.PercentLogUsed)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		ReplPendingXacts: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "repl_pending_xacts"),
-			"(ReplPendingXacts)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_repl_pending_xacts"),
+			"(Databases.ReplPendingXacts)",
+			[]string{"instance", "database"},
 			nil,
 		),
 		ReplTransRate: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "repl_trans_rate"),
-			"(ReplTransRate)",
-			[]string{"instance"},
+			prometheus.BuildFQName(Namespace, subsystem, "databases_repl_trans_rate"),
```

Comment:
What are typical values for this metric? This might also be a confusingly named counter

Reply:
Looks like all the `_rate` metrics are analogous to the `persec` ones. 
Changing them to Counters and tweaking names to be less confusing.